src/easy-sql-test.js

```diff
@@ -39,6 +43,10 @@ export default class EasySQLTest {
   }
 
   _executeStorProc(storProcName, args = {}, callback) {
+    if (!storProcName) {
```

Comment:
good point, added check for _query() function and updated unit tests accordingly